src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -64,37 +65,45 @@
 
 public class AndroidTargetConfiguration extends PosixTargetConfiguration {
 
+    private static final String ANDROID_TRIPLET = "aarch64-linux-android";
```

Comment:
shouldn't this be in Constants.Profile (full name)

Reply:
We have `Constants.PROFILE_ANDROID`, but that is only for `android`. The target triplet is created with `Triplet(Constants.Profiles.ANDROID)`, based on three constants, so probably we could have all possible triplets names as constants?

Reply:
@johanvos Your suggestion of doing `Constants.Profile (full name)` brings back this PR: https://github.com/gluonhq/substrate/pull/229, where `TripletProfile` was created precisely.

I guess this is a big refactoring that should be tackled in a follow-up PR?

For now, what if we just do:
```
private static final String ANDROID_TRIPLET = new Triplet(Constants.Profile.ANDROID).toString();
```